editor/src/components/canvas/commands/set-css-length-command.ts

```diff
@@ -75,45 +67,45 @@ export function setCssLengthProperty(
   }
 }
 
-export const runSetCssLengthProperty: CommandFunction<SetCssLengthProperty> = (
+export const runSetCssLengthProperty = (
```

Comment:
`runSetCssLengthProperty` is refactored not to read directly from the inline style prop